dlrover/python/diagnosis/diagnostician/metrics_collect_diagnostician.py

```diff
@@ -11,46 +11,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List
-
 from dlrover.python.common import env_utils
 from dlrover.python.diagnosis.common.constants import DiagnosisDataType
 from dlrover.python.diagnosis.common.diagnosis_data import WorkerTrainingMetric
-from dlrover.python.diagnosis.common.inference_chain import (
-    Inference,
-    InferenceAttribute,
-    InferenceDescription,
-    InferenceName,
-    InferenceOperator,
+from dlrover.python.diagnosis.common.diagnostician import (
+    DiagnosisObservation,
+    Diagnostician,
 )
 from dlrover.python.diagnosis.datacollector.xpu_timer_metric_collector import (
     XpuTimerMetricsCollector,
 )
 from dlrover.python.elastic_agent.master_client import MasterClient
 
 
-class MetricsCollectionOperator(InferenceOperator):
+class MetricsCollectDiagnostician(Diagnostician):
```

Comment:
Is it unnecessary to implement things like metric collectors as Diagnosticians? As long as the reported data meets the abstract design requirements, that's sufficient.